test/functional/__snapshots__/markdown-full.spec.js.snap

```diff
@@ -0,0 +1,64 @@
+exports[`createMarkdown outputs correct changelog 1`] = `
+"
+## Unreleased (2099-01-01)
+
+#### :rocket: New Feature
+* \`the-force-awakens\`, \`rogue-one\`
+  * [#7](https://github.com/lerna/lerna-changelog/pull/7) feat: that is not how the Force works!. ([@han-solo](https://github.com/han-solo))
+
+#### :nail_care: Enhancement
+* \`the-force-awakens\`, \`rogue-one\`
+  * [#7](https://github.com/lerna/lerna-changelog/pull/7) feat: that is not how the Force works!. ([@han-solo](https://github.com/han-solo))
+
+#### Committers: 1
+- Han Solo ([han-solo](https://github.com/han-solo))
+
+
```

Comment:
I thought it's a bit better to distinguish between tags / releases. If you think it's not important I can put only 1 break like back.